released/discovery_center/mission_3252/variables.tf

```diff
@@ -0,0 +1,164 @@
+######################################################################
+# Customer account setup
+######################################################################
+# subaccount
+variable "globalaccount" {
+  type        = string
+  description = "The globalaccount subdomain."
+  default     = "your globalaccount subdomain"
+}
+# subaccount
+variable "subaccount_name" {
+  type        = string
+  description = "The subaccount name."
+  default     = "DC Mission 3252 - Get Started with SAP BTP, Kyma runtime creating a Hello-World Function"
+}
+# Region
+variable "region" {
+  type        = string
+  description = "The region where the project account shall be created in."
+  default     = "eu30"
+}
+
+# CLI server
+variable "cli_server_url" {
+  type        = string
+  description = "The BTP CLI server URL."
+  default     = "https://cpcli.cf.eu10.hana.ondemand.com"
+}
+
+variable "subaccount_admins" {
+  type        = list(string)
+  description = "Defines the colleagues who are added to each subaccount as subaccount administrators."
+  default     = ["jane.doe@test.com", "john.doe@test.com"]
+}
+
+variable "subaccount_service_admins" {
+  type        = list(string)
+  description = "Defines the colleagues who are added to each subaccount as subaccount service administrators."
+  default     = ["jane.doe@test.com", "john.doe@test.com"]
+}
+
+
+
+###
+# Entitlements
+###
+variable "entitlements" {
```

Comment:
What are the entitlements used for? They are not reflected in the `main.tf`  and can probably be removed

Reply:
removed